### DISCUSSION TITLE
safe to upgrade 0.76.0-2

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#6852] [General] [answered]
Hello,

I saw there is a tag `v0.76.0-2` available in docker hub but no releases in github. Is it however a "standard" release ? the version is also unusual, that why Iḿ asking

Best

**Comment by tommoor** [comment, marked as answer]
The dash denotes a pre-release, for testing. Think of it like a beta release - there is more risk of new bugs

**Comment by d41ly** [comment]
Just upgraded, solved my issues with 0.76.0, no noticeable problems so far.
